SAKPaaS/src/app/components/user-card/user-card.component.scss

```diff
@@ -3,23 +3,101 @@
     display: none;
   }
   position: absolute;
-  top: 45px;
-  right: 30px;
-  border-radius: 10px;
+  top: 0;
+  right: 0;
+  border-radius: 0 0 0 10px;
+  width: 11%;
+  height: 14%;
+  box-shadow: none;
+  transition: all .3s;
   a {
     color: var(--greenblue);
     text-decoration: underline;
     cursor: pointer;
   }
-  mat-icon {
+  .settings-icon {
     display: none;
     float: right;
     padding-left: 10px;
     width: 34px;
   }
+  .progress-bar-container{
+    text-align: center;
+    color: var(--greenblue);
+    mat-progress-bar{
+      border-radius: 3px;
+    }
+    .level-name{
+      font-size: 12px;
+    }
+    .next-level{
+      font-size: 9px;
+    }
+  }
+
+  .expand-icon {
+    margin-top: 100%;
+    margin-bottom: 0;
+  }
+
+  .expand-icon:hover {
+    cursor: pointer;
+  }
+}
+
+.expand {
+  float: left;
+  width: 0%;
```

Comment:
There should be no width attribute here or else float won't work correctly.

Reply:
Ohne das width attribute bricht der content sehr schnell um und steht dann unter dem expand button

Reply:
Better under the button than behind the button. 😄
You could add a `min-width`-attribute to the container to prevent that.